stix/operations.py

```diff
@@ -48,163 +141,116 @@ def get_datetime(_epoch):
         raise ConnectorError(str(err))
 
 
-def indicator_ids(_indicators):
-    ind_id = []
-    for ind in _indicators:
-        ind_id.append(ind['id'])
-    return ind_id
+def get_epoch(_date):
+    try:
+        pattern = '%Y-%m-%dT%H:%M:%S.%fZ' if '.' in _date else '%Y-%m-%dT%H:%M:%SZ'
+        return int(time.mktime(time.strptime(_date, pattern)))
+    except Exception as err:
```

Comment:
there is no need to catch every exception and re-raise as ConnectorError unless you want to modify the error message or any other specific handling. The framework itself casts the final error into ConnectorError. Please remove all except blocks where we raise ConnectorError back